README.md

```diff
@@ -51,7 +51,6 @@ support today's systems.
 | **Feature**                 | **Status**
 |-                            |-
 | **CD-DA file codecs**       | Yes: Opus, OGG/Vorbus, MP3, FLAC, and WAV
-| **Pixel-perfect mode**      | Yes: `output=openglpp` or `output=texturepp`
```

Comment:
I was secretly hoping you would mention `integer_scaling` here, but we can do that later... 😄

Reply:
Will the new line do?

Reply:
You mean CR/LF? 😄 We can update this later, nevermind.

Reply:
Yes, nice, thanks.